### PR TITLE
readme improvements

### PR DESCRIPTION
**Pull request by bhageena**

### CODE REVIEW

File: shopify/README.md [hunk 1]
```diff
@@ -35,29 +35,29 @@ View list of past orders
...
-![Shopify products sold command](https://raw.githubusercontent.com/nimbella/command-sets/master/shopify/screenshots/productsSold.PNG)
+![Shopify products sold command](screenshots/productsSold.PNG)
 
 View product inventory
-Params: search_words (Comma deliminated list of text to filter by) EX: "textone, text two, text three four"
+Params: search_words (Comma delimited list of text to filter by) EX: "textone, text two, text three four"
 ```
 /nc shopify_inventory [<search_words>]
 ```
-![Shopify inventory command](https://raw.githubusercontent.com/nimbella/command-sets/master/shopify/screenshots/inventory.PNG)
```
Comment (satyarohith) [inline shopify/README.md]: The Shopify help command fetches readme directly from GitHub and relative URLs won't resolve on Mattermost/Slack.

File: corona_stats/README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Coronaviruses
+# Get live Coronavirus stats in Slack with Nimbella Commander
```
Comment (satyarohith) [inline corona_stats/README.md:1]:
```suggestion
# Coronavirus Stats

Get live COVID-19 stats in slack with Nimbella Commander. 
```
What do you think about this?

**Review by satyarohith — COMMENTED** [review]
For now, we can use relative URLs in READMEs except for Shopify.

**Comment by bhageena** [comment]
> For now, we can use relative URLs in READMEs except for Shopify.

I updated them for sake of consistency. Even absolute urls will not be a problem unless we move or rename the repo.

File: corona_stats/README.md (hunk 2 above)
Comment (bhageena) [reply corona_stats/README.md:1]: some crawlers use the first line as description, we can let it be a bit long as it gives complete sense.

**Review by niks3089 — APPROVED** [review]

File: corona_stats/README.md (hunk 2 above)
Comment (satyarohith) [reply corona_stats/README.md:1]: Okay

**Review by satyarohith — APPROVED** [review]
LGTM.

**Pull request merged**